src/main/webapp/app/exercises/shared/participation/participation.component.ts

```diff
@@ -350,4 +351,21 @@ export class ParticipationComponent implements OnInit, OnDestroy {
         }
         return repoUrl;
     };
+
+    /**
+     * Get the route for the exercise's scores page.
+     *
+     * @param exercise the exercise for which the scores route should be extracted
+     */
+    getScoresRoute(exercise: Exercise): string {
```

Comment:
I think this could return the route as an array instead of a string, so you don't have to add slashes everywhere and the navigation should still handle it fine (?)

Reply:
Will do!